ruby/lib/trailer_vote/api/issue/find.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'trailer_vote/api/composable/get'
+require 'trailer_vote/api/issue'
+
+module TrailerVote
+  module Api
+    class Configuration
+
+      # @return [TrailerVote::Api::Issue::Find] issues attached to the credentials
+      def issue
+        Issue::Find.new(configuration: self)
```

Comment:
Which `issue` would this find? Instead, you want to give the "Issue" api:
```suggestion
        Issue.new(configuration: self)
```